openhtf/io/frontend/prebuilt/styles/station.header.css

```diff
@@ -0,0 +1,61 @@
+.station-header {
+  margin: 0px 24px 0px 24px;
+  overflow: hidden;
+}
+
```

Comment:
you have two blank lines here but only 1 in station.css.. i don't know what style is commonly accepted, but they should probably be consistent at least.